core/src/main/kotlin/com/getstrm/pace/processing_platforms/postgres/PostgresTransformer.kt

```diff
@@ -35,6 +37,28 @@ object PostgresTransformer : ProcessingPlatformTransformer(PostgresRenderer) {
             )
         }
 
+    override fun hash(
+        field: DataPolicy.Field,
+        hash: DataPolicy.RuleSet.FieldTransform.Transform.Hash
+    ): Field<*> {
+        val dataType = field.toJooqField().dataType
+        return if (dataType.isNumeric) {
+            DSL.field(
+                "hashtextextended(cast({0} as varchar), ${if (hash.hasSeed()) hash.seed else 0})",
```

Comment:
It looks like only Postgres has seed support? Should we perhaps add (numeric) or prefix (string) the seed to the original value? On the other hand, the seed will be visible for anyone who can query the view, so it's not as if it adds a lot of security. If only one platform implements it though, it might be better to dop it for now?

Reply:
Snowflake has it as too, even though it doesn't add a lot of security, I'm going to keep it here.